### PR TITLE
update 4.1.1

### PR DESCRIPTION
**Pull request by Arishamays1**
## ydata-profiling 4.1.1 Update
[Jira Ticket](https://anaconda.atlassian.net/browse/PKG-1368)
[Upstream](https://github.com/FFY00/python-pyproject-metadata)
[Dependencies ](https://github.com/FFY00/python-pyproject-metadata/blob/main/setup.cfg)
# Changes
- Updated version number and `sha256`
- Updated pinnings on dependencies
- Added a specific pinning for `scipy` to version `1.9.3` due to upstream requirements for a successful `python 3.11` build

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -13,34 +13,41 @@ build:
...
-    - python >=3.7,<3.11
+    - python
     - pip
+    - setuptools
+    - wheel
   run:
-    - python >=3.7,<3.11
-    - scipy >=1.4.1,<1.10
+    - python
+    - scipy=1.9.3
```
Comment (ELundby45) [inline recipe/meta.yaml]: Curious why the scipy pinnings were changed?
 https://github.com/ydataai/ydata-profiling/blob/v4.1.1/requirements.txt#LL1-L1C20
 ```suggestion
    - scipy >=1.4.1,<1.10
```

File: recipe/meta.yaml [hunk 2]
```diff
@@ -61,11 +68,17 @@ test:
...
+  description: |
+    ydata-profiling primary goal is to provide a one-line Exploratory
+    Data Analysis (EDA) experience in a consistent and fast solution.
+    Like pandas df.describe() function, that is so handy, ydata-profiling
+    delivers an extended analysis of a DataFrame while allowing the data
+    analysis to be exported in different formats such as html and json.
   license: MIT
   license_family: MIT
   license_file: LICENSE
   doc_url: https://ydata-profiling.ydata.ai
```
Comment (ELundby45) [inline recipe/meta.yaml:80]:
```suggestion
  doc_url: https://ydata-profiling.ydata.ai
  dev_url: https://github.com/ydataai/ydata-profiling
```

**Review by ELundby45 — COMMENTED** [review]
v4.1.2 was released a couple of days ago. Not sure if we want to capture those bug fixes.
https://github.com/ydataai/ydata-profiling/releases/tag/v4.1.2

File: recipe/meta.yaml (hunk 1 above)
Comment (Arishamays1) [reply recipe/meta.yaml]: Hi Eric, there were a few issues raised regarding this in the [upstream](https://github.com/ydataai/ydata-profiling/issues/1284) where the requirements had a restriction of `scipy < 1.10` in which for our defaults, we did not have scipy version less than 1.10 was built for compatibility for python 3.11 which therefore caused build failures for this package that is needed for scikit-image 0.20.0

Comment (Arishamays1) [reply recipe/meta.yaml]: So scipy 1.9.3 had to be built for python 3.11 and uploaded to defaults specifically in support of this package

**Review by ELundby45 — APPROVED** [review]
The linter issues could be addressed, but overall lgtm

File: recipe/meta.yaml (hunk 1 above)
Comment (skupr-anaconda) [inline recipe/meta.yaml]: `scipy =1.9.3`

Comment (cbouss) [inline recipe/meta.yaml]:
```suggestion
    - scipy >=1.4.1,<1.10
```
See https://github.com/ydataai/ydata-profiling/blob/v4.1.1/requirements.txt#L1

File: recipe/meta.yaml [hunk 3]
```diff
@@ -13,34 +13,41 @@ build:
...
     - requests >=2.24.0,<2.29
+    # Progress bar
     - tqdm >=4.48.2,<4.65
-    - seaborn-base >=0.10.1,<0.13
+    - seaborn >=0.10.1,<0.13
     - multimethod >=1.4,<1.10
+    # metrics
     - statsmodels >=0.13.2,<0.14
     - typeguard >=2.13.2,<2.14
+    - imagehash
```
Comment (cbouss) [inline recipe/meta.yaml]:
```suggestion
    - imagehash ==4.3.1
```
See https://github.com/ydataai/ydata-profiling/blob/v4.1.1/requirements.txt#L24

File: recipe/meta.yaml [hunk 4]
```diff
@@ -13,34 +13,41 @@ build:
...
+    - pillow
     - ipywidgets >=7.5.1
```
Comment (cbouss) [inline recipe/meta.yaml]:
```suggestion
```
ipywidget is optional: https://github.com/ydataai/ydata-profiling/blob/v4.1.1/setup.py#L38

File: recipe/meta.yaml [hunk 5]
```diff
@@ -13,34 +13,41 @@ build:
...
   run_constrained:
     - jupyter-client >=5.3.4
```
Comment (cbouss) [inline recipe/meta.yaml:52]:
```suggestion
    - ipywidgets >=7.5.1
    - jupyter-client >=5.3.4
```
ipywidget is optional: https://github.com/ydataai/ydata-profiling/blob/v4.1.1/setup.py#L38

File: recipe/meta.yaml (hunk 1 above)
Comment (Arishamays1) [reply recipe/meta.yaml]: Thanks! Made the change.

Comment (Arishamays1) [reply recipe/meta.yaml]: Thanks Charles!

File: recipe/meta.yaml (hunk 3 above)
Comment (Arishamays1) [reply recipe/meta.yaml]: Thanks!

File: recipe/meta.yaml (hunk 5 above)
Comment (cbouss) [reply recipe/meta.yaml:52]: It should still be put in run_constrained

Comment (Arishamays1) [reply recipe/meta.yaml:52]: Thanks for the clarification! Pushed this change.

**Review by cbouss — APPROVED** [review]

**Review by skupr-anaconda — COMMENTED** [review]
LGTM

**Pull request merged**